services/ui_backend_service/data/db/tables/base.py

```diff
@@ -256,3 +256,29 @@ async def get_tags(self):
         except (Exception, psycopg2.DatabaseError) as error:
             self.db.logger.exception("Exception occured")
             return aiopg_exception_handling(error)
+
+    async def get_field_from(self, field: str, table: str, conditions: List[str] = [], values: List[str] = []):
+        sql_template = "SELECT {field_name} FROM {table_name} {conditions}"
+        select_sql = sql_template.format(
+            table_name=table,
+            field_name=field,
+            conditions=("WHERE {}".format(" AND ".join(conditions)) if conditions else "")
+        )
+
+        try:
+            with (
+                await self.db.pool.cursor(
+                    cursor_factory=psycopg2.extras.DictCursor
+                )
+            ) as cur:
+                await cur.execute(select_sql, values)
+
+                items = []
+                records = await cur.fetchall()
+                for record in records:
+                    items += record
+                cur.close()
+                return DBResponse(response_code=200, body=items)
+        except (Exception, psycopg2.DatabaseError) as error:
+            self.db.logger.exception("Exception occured")
+            return aiopg_exception_handling(error)
```

Comment:
instead of re-implementing the execution, could you use the `execute_sql` function that should be present for this scope?

Reply:
Hmm, it seems that execute_sql tries to serialize stuff before returning. Here i just want list of strings.

Reply:
I'm not currently utilizing those table objects here. I guess i should

Reply:
ah, true you want a simple list as a return. Disregard this then, should be fine as is.